test/reference.html

```diff
@@ -1050,6 +1053,13 @@
             this.innerHTML = '<span style="background:red; color: white; font-weight:bolder; padding-left:.5em; padding-right:.5em;">?</span>';
         }
     });
+    
+    // We've deferred the typeseting on startup. Do it explicitly now on each
+    // elements with a "mathjax" class.
+    $('.mathjax').each(function() { 
+        MathJax.Hub.Queue(["Typeset", MathJax.Hub, this]);
```

Comment:
if I comment this out, MathJax still auto-renders&mdash;I don't think this commit did anything